blocks/header-nav-chain-block/chains/header-nav-chain-block/default.jsx

```diff
@@ -111,14 +64,14 @@ export function PresentationalNav(props) {
 		primaryLogoAlt,
 	} = props;
 	return (
-		<StyledNav
+		<nav
 			id="main-nav"
-			className={navColorClass}
-			navBarBackground={backgroundColor}
-			navHeight={navHeight}
-			scrolled={scrolled}
-			// hard-coded to medium-breakpoint
-			breakpoint={mediumBreakpoint}
+			className={`${BLOCK_CLASS_NAME} ${navColorClass}`}
+			// navBarBackground={backgroundColor}
```

Comment:
This is a styled component prop - can this be converted as part of this phase?

Reply:
Covering this in https://arcpublishing.atlassian.net/browse/TMEDIA-888.....sort of leaving the props in there as reminders